src/Components/Blueprints/BlueprintsSideBar.tsx

```diff
@@ -52,7 +54,25 @@ const BlueprintsSidebar = ({
   );
 
   if (blueprints === undefined || blueprints?.length === 0) {
-    return emptyBlueprints;
+    if (filter.length === 0) {
+      return emptyBlueprints;
+    } else {
+      return (
+        <>
+          <Stack hasGutter>
+            <StackItem>
+              <SearchInput
+                placeholder="Search by name or description"
+                value={filter}
+                onChange={(_event, value) => onChange(value)}
+                onClear={() => onChange('')}
+              />
+            </StackItem>
+            <StackItem>No Blueprints found</StackItem>
```

Comment:
How about using the empty state design ?
![Screenshot 2024-01-29 at 14 46 53](https://github.com/RedHatInsights/image-builder-frontend/assets/11807069/6c072ab0-a0a6-4025-a2fe-4c54c5e3885e)